### ISSUE TITLE
Forget Password Link is not working 

### ISSUE THREAD

**Issue by 221996Vivekanand** [rajashree27/MyFitnessPal-Clone#51]
Step to produce this issue 
1. Land to the home page of the screen by clicking into the link
2. click on login link on right top of the screen. Once clicked you land-up to login page
3. After that click on the Forgot Password? link 
That link (Forgot Password?) is not working at all
<img width="605" alt="image" src="https://user-images.githubusercontent.com/59307548/190878416-2e5b2069-fcb2-4ccc-ac0e-3b2ba2fdaa47.png">

**Comment by 221996Vivekanand** [comment]
Another Issue Just in Comment instead of raising the Multiples
![image](https://user-images.githubusercontent.com/59307548/190878508-52a56537-8ed8-469e-87ca-007dca6a834f.png)
there is no validation on input field, user can gives any types of data, then the purpose of this asking information is useless

**Comment by 221996Vivekanand** [comment]
<img width="756" alt="image" src="https://user-images.githubusercontent.com/59307548/190878707-6a8f6959-ec23-4b6b-8880-5cd048afdc3f.png">
1 on the signup page: in the password field (password length 10), but i have only put 4 length of password still will able to login successfully 
**2. Console log should not be in the code
3. LogOut is not working 
4. Settings is also not working**

**Comment by 221996Vivekanand** [comment]
1. if you click on premium you land-up on the premium route but user information lost 
2. Community route not working
3. Reports Route not working
4. login/success route giving 403 still user information in the page means user look like login into the application
5. Basic links should be work as expected, because at time of interview may be interviewer can click on the basics link

**Comment by 221996Vivekanand** [comment]
**This issue is related to the application**
1. Responsiveness is looks good 
2. Backend guys has not worked properly there are several places we have seen the backend call with console log and error on the inspect page
3. Structure of the application is ok
4. look an feel is also ok

**Comment by 221996Vivekanand** [comment]
<img width="1716" alt="image" src="https://user-images.githubusercontent.com/59307548/190891507-76654898-8134-4dca-b2fb-be873011b037.png">
If i dived the application based on feature then all the navigation link are going to refer one feature
1. My Home looks fine, still there is some issue on this page, non of the link is working as expected
2. Food look fine 
3. Exercise look fine
4. Reports not working as expected
5. community not working as expected 
6. Premium looks fine but if you click then its look like different page navigation, why bcz user information is lost
Important Note: look like in this project only png file is attached:
for example if you navigate to food section, just observe the calendar, you will not be able to select past or future date, nothing clickable it's just look like png or jpg file attached
